grpc/client.go

```diff
@@ -282,17 +284,30 @@ func (c *Client) convertToMethodInfo(fdset *descriptorpb.FileDescriptorSet) ([]M
 				appendMethodInfo(fd, sd, md)
 			}
 		}
+
+		s := stack.New()
```

Comment:
I would just make this a slice and will `append` on `push` and on pop I will `message = s[len(s)-1]; s = s[:len(s)-1]` 

This is all we need in practice. This also will help with the type safety

Reply:
Perfect! 🆙
Better than that, impossible! 🚀
Thank you for your help!